netci.groovy

```diff
@@ -73,27 +73,28 @@ class WcfUtilities
             return 
         }
 
-        wcfRepoSyncServiceCount++
+        wcfPRServiceCount++
```

Comment:
Thanks @iamjasonp for the explanation. This is very helpful to understand what I can expect!